src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -37,6 +37,7 @@ When your data reaches high volumes, Sentry begins to automatically prioritize y
 | Latest Release       | Health Checks |
 | Dev Environments     |               |
 | Starred Transactions |               |
+| Low Volume Projects  |               |
```

Comment:
I think we need an asterisk (*) here to mention that this one differently from the others can't be turned off. You can maybe add it below in the paragraph that explains the bias.